src/fal/fal_script.py

```diff
@@ -23,12 +23,18 @@ def exec(self, context, faldbt: FalDbt):
 
         # Enable local imports
         local_path = str(self.path.parent)
-        sys.path.append(local_path)
+        try:
```

Comment:
We're not excepting anything?

Reply:
No, we use the `finally` to ALWAYS pop the local_path from path